resources/event/event.controller.js

```diff
@@ -0,0 +1,29 @@
+const Event = require("./event.model");
+const ScrapeEvents = require("../../utils/webScraper");
+
+const addScrapedEvent = async () => {
+  const scrappedEventsArray = await ScrapeEvents;
+  await scrappedEventsArray.forEach(async (event) => {
```

Comment:
Instead of looping through the event and inserting one after each, this isn't efficient enough instead let's use bulk insert without looping through the event array for better perfomance

Reply:
I agree. We will refactor to use `insertMany()`.